apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -226,14 +232,34 @@ suite('conversation.js >', function() {
     this.sinon.stub(MessageManager, 'on');
     this.sinon.stub(Compose, 'on');
     this.sinon.stub(Compose, 'off');
+    this.sinon.stub(Compose, 'init');
     this.sinon.stub(Threads, 'get');
+    this.sinon.stub(MozSettingsClient, 'mmsSizeLimitation');
+    this.sinon.stub(MozSettingsClient, 'maxConcatenatedMessages');
     this.sinon.useFakeTimers();
 
     this.sinon.stub(ActivityClient);
 
     ConversationView.recipients = null;
     ConversationView._isReady = false;
-    ConversationView.init();
+    mmsSizeLimitation = 295 * 1024;
```

Comment:
__optional nit:__ could you please add new line before ```mmsSizeLimitation = 295 * 1024;``` and ```ConversationView.init()``` just to make it a bit easier to read.

__question:__ Also what about making ```Conversation.init().then(done, done)``` as the last statement in ```setup``` and move asserts to dedicated test below (```test('View is ready after init()', ...)```), so no need in verifying private field ```assert.isTrue(ConversationView._isReady);``` ;)?